_sass/base/_archive_colors.scss

```diff
@@ -0,0 +1,6 @@
+// Colors for the archive version of the site
+$mainBrandColor: #466BB0;
+$secondBrandColor: #CEAF0C;
```

Comment:
let's not change the colors it's confusing for users (and not good for the brand)
as long as each page has that "archive" marking it's good